### PR TITLE
Release 0.5.2

### PR DESCRIPTION
**Pull request by pablolh**
### Added
- Variable declaration and initialization in cqasm v3 parser
- New types in v3: bool, int, float, axis
- Arrays in v3

### Changed

- v3x::AnalyzeTreeGenAstVisitor is an instance of tree-gen generated visitor base class

### Removed

- msvc-runtime pip dependency on Windows

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,21 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [ 0.5.2 ] - [ 2024-01-18 ]
+
+### Added
+- Variable declaration and initialization in cqasm v3 parser
```
Comment (rturrado) [inline CHANGELOG.md:9]: I may resume these 3 comments maybe just in:
- cQasm 3.0 parser: implemented variable declaration and initialization for `bool`, `int`, and `float` (simple and array), and `axis` types.

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,21 @@
...
+- New types in v3: bool, int, float, axis
+- Arrays in v3
+
+### Changed
+
```
Comment (rturrado) [inline CHANGELOG.md:14]: I'd remove this blank line.

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,21 @@
...
+- v3x::AnalyzeTreeGenAstVisitor is an instance of tree-gen generated visitor base class
```
Comment (rturrado) [inline CHANGELOG.md:15]: And use `v3x::AnalyzeTreeGenAstVisitor` and `tree-gen`.

File: CHANGELOG.md [hunk 4]
```diff
@@ -3,6 +3,21 @@
...
+
+### Removed
+
```
Comment (rturrado) [inline CHANGELOG.md:18]: I'd remove this blank line.

File: CHANGELOG.md [hunk 5]
```diff
@@ -3,6 +3,21 @@
...
+- msvc-runtime pip dependency on Windows
```
Comment (rturrado) [inline CHANGELOG.md:19]: And use `msvc-runtime`.

**Review by rturrado — APPROVED** [review]
I just wrote some comments, but I honestly wouldn't run GitHub Actions again in order to address them haha

Thanks, Pablo!

File: CHANGELOG.md (hunk 3 above)
Comment (pablolh) [reply CHANGELOG.md:15]: ? I don't understand

File: CHANGELOG.md (hunk 4 above)
Comment (pablolh) [reply CHANGELOG.md:18]: Apparently we do use blank lines so I've added the missing one instead

File: CHANGELOG.md (hunk 5 above)
Comment (pablolh) [reply CHANGELOG.md:19]: Oh, I understand now

**Pull request merged**